src/NuGet.Core/NuGet.Protocol.Core.v3/LegacyFeed/V2FeedParser.cs

```diff
@@ -133,11 +133,11 @@ public V2FeedParser(HttpSource httpSource, string baseAddress, PackageSource sou
                 throw new ArgumentNullException(nameof(token));
             }
 
-            var uri = String.Format(
+            var uri = string.Format(
                 CultureInfo.InvariantCulture,
                 GetPackagesFormat,
-                package.Id,
-                package.Version.ToNormalizedString());
+                WebUtility.UrlEncode(package.Id),
```

Comment:
Just a thought, why use `System.Net.WebUtility` when you can use a more common type: `System.Uri.EscapeDataString`